oxide/lib_test.go

```diff
@@ -144,3 +155,58 @@ func Test_buildRequest(t *testing.T) {
 		})
 	}
 }
+
+func Test_Client(t *testing.T) {
+	tt := map[string]struct {
+		config    *Config
+		env       map[string]string
+		wantError bool
```

Comment:
To test that Client has been created like we expect it to let's add the following to this struct:
```go
want string
```